app/controllers/application_controller.rb

```diff
@@ -1,2 +1,21 @@
 class ApplicationController < ActionController::Base
```

Comment:
Great to use the application controller so that we can use these instances in our application layouts views on every page